src/Bootstrap/src/Autoconfig/Steeltoe.Bootstrap.Autoconfig.csproj

```diff
@@ -13,30 +13,29 @@
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\..\..\Common\src\Common\Steeltoe.Common.csproj" />
-    <ProjectReference Include="..\..\..\Configuration\src\CloudFoundryCore\Steeltoe.Extensions.Configuration.CloudFoundryCore.csproj" PrivateAssets="All" />
-    <ProjectReference Include="..\..\..\Configuration\src\ConfigServerCore\Steeltoe.Extensions.Configuration.ConfigServerCore.csproj" PrivateAssets="All" />
-    <ProjectReference Include="..\..\..\Configuration\src\KubernetesCore\Steeltoe.Extensions.Configuration.KubernetesCore.csproj" PrivateAssets="All" />
-    <ProjectReference Include="..\..\..\Configuration\src\RandomValueBase\Steeltoe.Extensions.Configuration.RandomValueBase.csproj" PrivateAssets="All" />
-    <ProjectReference Include="..\..\..\Configuration\src\PlaceholderCore\Steeltoe.Extensions.Configuration.PlaceholderCore.csproj" PrivateAssets="All" />
+    <ProjectReference Include="..\..\..\Configuration\src\CloudFoundry\Steeltoe.Extensions.Configuration.CloudFoundry.csproj" PrivateAssets="All" />
+    <ProjectReference Include="..\..\..\Configuration\src\ConfigServer\Steeltoe.Extensions.Configuration.ConfigServer.csproj" PrivateAssets="All" />
+    <ProjectReference Include="..\..\..\Configuration\src\Kubernetes\Steeltoe.Extensions.Configuration.Kubernetes.csproj" PrivateAssets="All" />
+    <ProjectReference Include="..\..\..\Configuration\src\RandomValue\Steeltoe.Extensions.Configuration.RandomValue.csproj" PrivateAssets="All" />
+    <ProjectReference Include="..\..\..\Configuration\src\Placeholder\Steeltoe.Extensions.Configuration.Placeholder.csproj" PrivateAssets="All" />
     <ProjectReference Include="..\..\..\Connectors\src\Connector.EF6Core\Steeltoe.Connector.EF6Core.csproj" PrivateAssets="All" />
```

Comment:
I don't think EF6Core adds any NuGet references, it could potentially be merged with the Connector package. Comments or opinions appreciated here. The `EFCore` project should probably not lose `Core` since it's for EntityFrameworkCore.. That one does bring in NuGet references so probably should not be merged

Reply:
Alright, I'm going to push the changes: `EF6Core` -> `EntityFramework6` and `EFCore` -> `EntityFrameworkCore` for now, and we can revisit these packages during the Connector refactoring work if it becomes clearer by then

Reply:
I'm fine with the renames, just wondering why we need to keep EntityFramework6 in?

Reply:
Entity Framework 6 is still supported and targets netstandard2.1, so should still work on .NET 6+. Our package doesn't get used a whole lot, but since it still works and is supported, I'm inclined to leave it alone. We could merge it into the connector package, but I'm thinking we might as well leave it standalone, at least until we come back for the connector refactor work